File: alisaurus.c
#include <stdio.h>

int main(){

  int t;
  int x;

  // Read in number of scenarios
  scanf("%d", &t);

  // Loop over each scenario
  for(int i = 0; i < t; i++){
    // Read in the number of alisauruses
    scanf("%d", &x);

    // If they can evenly divide the circle, they will attack
    if(360 % x == 0){
        printf("Attack!\n");
    }
    else{
        printf("No go.\n");
    }

  }

}

File: alisaurus.py
# Read in number of scenarios
t = int(input())

# Loop over scenarios
for i in range(t):
    # Read in number of Alisauruses
    x = int(input())

    # If they split the circle evenly, they can attack!
    if(360 % x == 0):
        print("Attack!");
    else:
        print("No go.");
